src/Dapr.Client/DaprClient.cs

```diff
@@ -1022,6 +1020,64 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             string workflowComponent,
             CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Attempt to raise an event the given workflow with response indicating success.
+        /// </summary>
+        /// <param name="instanceId">Identifier of the specific run.</param>
+        /// <param name="workflowComponent">The component to interface with.</param>
+        /// <param name="eventName">Name of the event to raise.</param>
+        /// <param name="eventData">The JSON-serializable event payload to include in the raised event.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="Task" /> that will complete when the raise event operation has been scheduled. If the wrapped value is true the operation suceeded.</returns>
+        [Obsolete("This API is currently not stable as it is in the Alpha stage. This attribute will be removed once it is stable.")]
+        public abstract Task RaiseWorkflowEventAsync(
+            string instanceId,
+            string workflowComponent,
+            string eventName,
+            Object eventData,
```

Comment:
Let's set the default value to null since not all raised events need to have data payloads. Also, C# convention is to prefer `object` (lowercase/keyword) instead of `Object` (capitalized).
```suggestion
            object eventData = null,
```